janusgraph-core/src/main/java/org/janusgraph/graphdb/query/ElementQuery.java

```diff
@@ -25,7 +26,7 @@
  * @author Matthias Broecheler (me@matthiasb.com)
  */
 
-public interface ElementQuery<R extends JanusGraphElement,B extends BackendQuery<B>> extends Query {
```

Comment:
Question: what is the behavior of this change? I'm just curious.

Reply:
There is no behavior change. `ElementQuery` has two concrete subclasses, `GraphCentricQuery` and `VertexCentricQuery`, both of which have implemented the `ProfileObservable` interface. Thus I made `ElementQuery` itself extend `ProfileObservable`. It's just a code cleanup.

Not sure but I think I made this change in https://github.com/JanusGraph/janusgraph/pull/2289 to avoid explicit casting.